### ISSUE TITLE
Common Features should be run first

### ISSUE THREAD

**Issue by vrutkovs** [Containers-Testing-Framework/ctf-cli#15]
Starting tests if general features don't pass is pointless, currently those features are queued at the end.

We should either start behave twice (once for common Docker features and then other tests) or make sure common-features are being run first

**Comment by rwngwn** [comment]
I don't think that running behave multiple time is a good idea. Maybe we should introduce some dependency decorator and then just skip rest of tests using something like

``` python
for scenario in feature.scenarios:
    scenario.mark_skipped  
```

Also test steps/features ordering seems for me more like behave stuff that should be fixed in upstream and reused in this framework rather than implemented here in a hacky way.
